sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -544,6 +547,36 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
           },
         ],
       },
+      {
+        path: AppRoutes.ApplicationDetail,
```

Comment:
Are we missing the `allowOnlyBCPublic` here?

Reply:
Yes is is missing and required here.